itou/metabase/db.py

```diff
@@ -144,6 +146,132 @@ def build_final_tables():
         print("Done.")
 
 
+def create_unversioned_tables_if_needed():
+    """
+    Unfortunately some tables are not versioned yet as they are still managed manually by our data analysts.
+    This becomes a problem when trying to run all requests on a local empty database.
+    The present function creates these unversioned tables without any content, at least now all the requests
+    can complete locally and we have a good visibility of how many tables are left to be automated.
+    """
+    with MetabaseDatabaseCursor() as (cur, conn):
+        create_table_sql_requests = """
+            CREATE TABLE IF NOT EXISTS "commune_gps" (
+                "code_insee" varchar(255),
+                "nom_commune" varchar(255),
+                "code_postal" varchar(255),
+                "latitude" numeric(9,6),
+                "longitude" numeric(9,6)
+            );
+
+            CREATE TABLE IF NOT EXISTS "constantes" (
+                "annee_en_cours" float8
+            );
```

Comment:
à quoi sert cette table pour ma culture ? On met quoi dans "annee_en_cours" ?

Reply:
La table `constantes  contient des années liées aux annexes financières. Je pense qu'on est pas obligés de créer une table parce que `constantes` est directement générée à partir des données de l'ASP. Je pense que cette table a été créée par erreur et jamais effacée.

Reply:
Lol bien vu Yannick, j'ai vu un fantôme. Une requête a du casser, j'ai noté ce nom `constantes` dedans et vu qu'il y avait une table avec ce nom en prod, du coup j'ai immédiatemment assumé que c'était la cause du plantage, alors qu'en fait non la requête devait planter sur une autre des tables non versionnées.

Je la vire du code et je la supprime en prod aussi. 🔪

Reply:
Parfait